blog.html

```diff
@@ -22,6 +22,146 @@
+                    The ability to migrate Kafka clusters from a ZooKeeper metadata system to a KRaft metadata system is
+                    now ready for usage in production environments. See the ZooKeeper to KRaft migration
+                    <a href="https://kafka.apache.org/documentation/#kraft_zk_migration">operations documentation</a> for
+                    details. Note that support for JBOD is still not available for KRaft clusters, therefore clusters
+                    utilizing JBOD can not be migrated. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-858%3A+Handle+JBOD+broker+disk+failure+in+KRaft">KIP-858</a>
+                    for details regarding KRaft and JBOD.
+                </p>
+                <p>Support for Delegation Tokens in KRaft (<a href="https://issues.apache.org/jira/browse/KAFKA-15219">KAFKA-15219</a>) was completed in 3.6, further reducing the gap of features between ZooKeeper-based Kafka clusters and KRaft. Migration of delegation tokens from ZooKeeper to KRaft is also included in 3.6.</p>
+                <p><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-405%3A+Kafka+Tiered+Storage">Tiered Storage</a> is an early access feature. It is currently only suitable for testing in non production environments. See the <a href="https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Tiered+Storage+Early+Access+Release+Notes">Early Access Release Notes</a> for more details.</p>
+
+                <p><i>Note: ZooKeeper is marked as deprecated since 3.5.0 release. ZooKeeper is planned to be removed in Apache Kafka 4.0. For more information, please see the documentation for <a href="/documentation#zk_depr">ZooKeeper Deprecation</a></i><p>
+                <h3>Kafka Broker, Controller, Producer, Consumer and Admin Client</h3>
+                <ul>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-405%3A+Kafka+Tiered+Storage">KIP-405</a>:
+                        Kafka Tiered Storage (Early Access): </b><br>Introduces Tiered Storage to Kafka. Note that this
+                        is an early access feature only advised for use in non-production environments (see the <a
+                                href="https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Tiered+Storage+Early+Access+Release+Notes">early
+                            access notes</a> for more information). This feature provides a separation of computation
+                        and storage in the broker for pluggable storage tiering natively in Kafka Tiered Storage brings
+                        a seamless extension of storage to remote objects with minimal operational changes.
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-405%3A+Kafka+Tiered+Storage">KIP-890</a>:
+                        Transactions Server Side Defense (Part 1): </b><br>Hanging transactions can negatively impact
```

Comment:
We should add a note that we missed the fix for offset commits. That will be addressed in the next release.
Something like. 

In 3.6.0, transaction verification will prevent hanging transactions on data partitions. In the next release, transactional offset commits will also be covered.